### DISCUSSION TITLE
Error:net

### DISCUSSION THREAD

**Discussion by dm0n7** [KevinBytesTheDust/CVAmp#308] [Q&A]
Hello, I dont know whats wrong with my version of the program. A few days ago I was using it with normally but later, this error started to appear "TWITCH Instance 1 died: Error:net::ERR_TUNNEL_CONNECTIO... Please see cvamp.log."

Im using the v0.7.1
Windows 10 
Chrome version 129.0.6668.71
Im currently using 100 proxies (premium version) from webshare.io

Thanks for the help. Have a great one!

**Comment by KevinBytesTheDust** [comment]
Did you resolve your problem?
